src/bootstrap/index.js

```diff
@@ -3,12 +3,21 @@ const config = require("./lib/config");
 const logger = require("./lib/logger");
 const middleware = require("./middleware");
 const redisClient = require("./lib/redis-client");
+const {
+  configure: configureOverloadProtection,
+} = require("./lib/overload-protection");
 
 const setup = (
   options = {
     middlewareSetupFn: undefined,
+    overloadProtection: undefined,
   },
 ) => {
+  const protect = require("overload-protection")(
+    "express",
+    configureOverloadProtection(options.overloadProtection),
```

Comment:
This doesn't match the `...config.get("overloadProtection")` approached used further down. 

I think there's functionality we're not using in that approach, but I'm not sure its currently worth investing time into replicating the feature parity. Happy to be challenged on that now, or at a later date.